credly/views.py

```diff
@@ -592,10 +593,16 @@ def claim_badge(request, badge_id):
         url = CREDLY_API_URL + 'me/claimable_badges/claim/' + badge_id
         result = requests.post(url, params=params, headers=headers, data=data)
 
+        if result.status_code is not 200:
+            print "failed to POST a badge claim to Credly:"
+            print url, params, headers, data
+            print "Result:"
+            print result.text
+
     except HttpClientError as error:
         print error
         pass
 
     return json_response(res, {
         'result': result.text
-    })
+    }, result.status_code)
```

Comment:
this was necessary because 404s from Credly were being sent on as 200s from the Teach-API, which is of course not right at all.